README.md

```diff
@@ -49,4 +49,15 @@ some ticket. Then you have:
  5. Wait for architect to merge (architect may ask additional changes)
  6. After merge to `master` ask issue author to close the ticket
  
+Git guidelines:
+ - start all commit message with tiket number, e.g.: `#1 - some message`
+ - try to describe shortly your changes in commit message, long description
+ should be provided in PR body
+ - name your barnches same as ticket number, e.g. `1` branch for #1 ticket
```

Comment:
@g4s8 can we also allow for naming branches with `1-some-small-desc`? I really don't like the usage of pure numbers because when switching regularly between branches, it's hard to find back what we are looking for.